### DISCUSSION TITLE
Cubesatsim won't start when powered by batteries

### DISCUSSION THREAD

**Discussion by yixinyxl** [alanbjohnston/CubeSatSim#160] [Q&A]
When I remove the RBF, I can see red light on the 5V regulator, yellow blink on the rasberry pi 2 w, and the red power light on the STM32F board, but the green, blue lights on the main board are always off, no telemetry is transmitted.  Here is the link of the log after plugged the usb in the Pi's USB port with battery board connected to the main board: https://github.com/ihsinliu/CubeSatSim/blob/main/log%207.27.2022%201825.txt. The voltage measured on the battery board is around 4.1v before I started the system.

Edit1: Here is a 30 seconds video starting from unplugging the RBF: https://drive.google.com/file/d/1iZ5Nt2L-dwBMdtAfyndrEQNfocjcdEt2/view?usp=sharing

**Comment by alanbjohnston** [comment]
Hi! Sorry for the delay in replying - I’ve been traveling.  

It’s not clear what the problem is. The log file you posted looks like it is running OK.  It shows the BPF is detected so it will transmit. The battery voltage is 3.8 V when it is running which is above the 3.7 V threshold. 

Did the green LED test OK when you did this step? https://github.com/alanbjohnston/CubeSatSim/wiki/4.-Main-Board-2#power-led-test

The LEDs are controlled by some Python code. You could try reinstalling it by typing these 4 commands. Copy and paste any output.

cd 
cd pi-power-button
./script/install

reboot and see if you see anything on the green LED. 

If you could also share the log from the transmit Python code by typing:

cd 
CubeSatSim/log -r

You should also listen closely around 434.9 MHz as it powers up to see if you hear CW (Morse code) for a few seconds. 

Good luck!
Alan

**Comment by Boovtech** [comment]
Did you set up a ground station or use a radio just to listen and see what it is transmitting.? Maybe it is - as suggested simply that the LEDs are not responding.

73
Virginia
